lib.js

```diff
@@ -0,0 +1,64 @@
+/* eslint-disable no-use-before-define */
+const myLibrary = [];
```

Comment:
### [OPTIONAL]

- [ ] It will be great to save your library array to the local storage. This ensures that your data is persistent and does not disappear once the application is refreshed. To learn more about local storage you can use this [article](https://www.w3schools.com/jsref/prop_win_localstorage.asp) ⬇️ 
<img width="848" alt="localstorage_stretch_requirement" src="https://user-images.githubusercontent.com/32361963/128160599-f3cef82b-8f5d-4f27-ad78-27447e4e1187.png">